biaspaltsau_a/14.Ansible.Workshop/hosts.yaml

```diff
@@ -0,0 +1,15 @@
+all_hosts:
+      wars:
+        ansible_ssh_common_args: '-o ProxyCommand="ssh -W %h:%p -q jump_sa@178.124.206.53"'
+      hosts:
+        redmine:
+          ansible_host: 192.168.201.2
+          ansible_user: root
+
+
+
+
+jump:
```

Comment:
This is needlessly, please remove.
in this play we are doing nothing on jump server

Reply:
corrected